rs-week1/src/App.tsx

```diff
@@ -1,34 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import Header from './components/header/Header';
+import FormPage from './pages/Form/FormPage';
```

Comment:
```suggestion
import FormPage from './pages';
```
создай в папке "pages"  файл 'index.ts' и сделай реэкспорт 
вот такую строку : export * from '.Form/FormPage';